test/fixtures/todo/json5/serialization/object.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = [
+  {
+    name: 'object with numeric literal as key',
+    value: { 42: true },
+    serialized: '{42:true}'
+  },
+  {
+    name: 'object with unquoted keys',
```

Comment:
@nechaido, yes, and these JSON5 tests are imported while you are creating new tests, in my opinion, it is better to name them correctly.